core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,670 @@
-    border-radius: 4px;
-    color: white;
-    letter-spacing: 0.7px;
-    width: 33%;
-  }
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor:hover {
-    background-color: #3d9991;
-    text-decoration: none;
-  }
-  .oppia-about-page-container .oppia-about-text-align {
+  .oppia-about-section {
+    background-color: #fff;
+    clear: both;
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+    max-width: 100%;
+  }
+  .oppia-about-fixed-section {
+    justify-content: center;
+    width: 1440px;
+  }
+  .oppia-about-page-section-one {
+    background-color: #F4F4F4;
+    height: 60vh;
+    padding-top: 0;
+  }
+  .oppia-about-page-section-one .oppia-about-page-section-one-right {
+    float: right;
+    height: 60vh;
+    width: 50%;
+  }
+  .oppia-about-page-section-one .oppia-about-page-section-one-left {
+    float: right;
+    width: 635px;
+  }
+  .oppia-about-page-section-one .oppia-about-title {
+    font-size: 45px;
+    padding-top: 10vh;
+  }
+  .oppia-about-page-section-one .oppia-about-content {
+    font-size: 23px;
+    font-weight: 400;
+    padding-right: 4vw;
+  }
+  .oppia-about-page-section-one .oppia-about-study-image {
+    float: right;
+  }
+  .oppia-about-page-section-one .oppia-about-study-image img {
+    height: 60vh;
+  }
+  .oppia-about-page-section-two {
+    height: auto;
+    padding-bottom: 5vh;
+    width: 100%;
+  }
+  .oppia-about-page-section-two .oppia-section-two-content {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+  .oppia-about-page-section-two .oppia-about-title {
+    padding-top: 6.5vh;
     text-align: center;
   }
-  .oppia-about-page-container .oppia-about-three-cols-layout {
-    display: inline-block;
-    margin-right: 20px;
-    width: -webkit-calc((100% - 60px) / 3);
-    width: -moz-calc((100% - 60px) / 3);
-    width: -o-calc((100% - 60px) / 3);
-    width: calc((100% - 60px) / 3);
+  .about-page .oppia-about-page-title {
+    color: #00424A;
+    font-family: "Capriola", Arial, sans-serif;
+    font-size: 40px;
+    font-weight: normal;
+    margin: 0;
+    width: 30vw;
   }
-  .oppia-about-page-container .oppia-about-three-cols-layout:last-of-type {
-    margin-right: 0;
+  .oppia-about-page-section-two .oppia-about-content {
+    text-align: center;
+    width: 768px;
   }
-  @media only screen and (max-width: 500px) {
-    .oppia-about-page-container .oppia-about-three-cols-layout {
-      width: 100%;
-    }
+  .about-page .oppia-about-page-content {
+    color: #333;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 20px;
+    line-height: 1.5;
+    margin: 0;
+    padding-bottom: 5vh;
+    padding-top: 4vh;
+    width: 35vw;
   }
-  .oppia-about-page-container .oppia-about-three-cols-feature,
-  .oppia-about-page-container .oppia-about-three-cols-feature p {
+  .oppia-about-page-section-two .oppia-about-icons {
+    display: flex;
+    justify-content: center;
+    max-width: 1440px;
```

Comment:
Why icons have max-width 1440px?

Reply:
yes all the icons need to be placed in fixed width of 1440px according to the mocks.